src/api/app/views/webui/request/beta_show_tabs/_conversation.html.haml

```diff
@@ -7,22 +7,20 @@
         %i No description set
 .row
   .col-md-4.order-md-2.order-sm-1.mb-4#side-links
-    - if accepted_reviews.present? || declined_reviews.present? || open_reviews.present?
+    - if request_reviews.present?
       .d-flex.justify-content-between
         %h4
-          Reviews
-        = render AddReviewDropdownComponent.new(bs_request: bs_request, user: User.session,
-                                                can_add_reviews: can_add_reviews,
-                                                my_open_reviews: my_open_reviews)
+          Reviewers
+        %button.btn.btn-link{ data: { toggle: 'modal', target: '#add-reviewer-modal' }, title: 'Ask for a Review' }
```

Comment:
The alignment of this button with the heading text is off:

![alignment-is-off](https://user-images.githubusercontent.com/1102934/196468846-fc68b99a-5df8-4cc8-8ff1-fbe536cb6a36.png)

Reply:
As it was before unfortunately
![offset](https://user-images.githubusercontent.com/7080830/196476659-2903904c-633f-4029-a27d-88f96741930c.png)

I'll try to get it adjusted anyway

Reply:
It's a workaround more than a fix --> https://github.com/openSUSE/open-build-service/pull/13227/commits/86f0dea7b4f8a60753ea769fe60fd9ed6983d9bf

Reply:
Why it's a workaround and not a fix? :thinking:

Reply:
Because we are adding a specific margin-top value (`mt-2`) to the `<h4>` to vertically align it to the button, but if something in that context will change (button top padding or margin, etc) the alignment will need another round of adjustment.
But it can also be just a matter of definition, it actually fixes it :slightly_smiling_face:

Reply:
Okay, I understand it now. Thanks!

Reply:
As discussed offline, `m-0` to the `h4` and `.align-items-center` to the container does the fix as well, deleting the `h4` bottom margin default, but moving it up to the header container level.